### ISSUE TITLE
Electron/node packages no longer hosted by Atom?

### ISSUE THREAD

**Issue by kistlin** [cmake-js/cmake-js#296]
Hello,

it seems like `Atom` is no longer actively developed and `cmake-js` is currently fetching packages from their server.

```
We are archiving Atom and all projects under the Atom organization for an official sunset on December 15, 2022.
```
https://github.blog/2022-06-08-sunsetting-atom/

Now I get an error when trying to build.
```
stderr info
stderr  DIST Downloading distribution files.

stderr http
stderr  DIST    - https://atom.io/download/atom-shell/v9.2.1/node-v9.2.1.tar.gz

stderr ERR!
stderr  OMG incorrect header check
ERR! OMG incorrect header check
```

There is a variable `ELECTRON_MIRROR` to overwrite the default url. This requires that both `node` and `electron` are hosted on the same server.

The default should be changed and maybe two separate variables for each package, as `node` for example only hosts `node`.
https://nodejs.org/dist

Does someone else see the same problem? Not that I am on a completely wrong track.

**Comment by kistlin** [comment]
Ok there are more variables, forget about that.

But it tries to fetch `node` from `ELECTRON_MIRROR` instead of `NODE_MIRROR`

I quickly tried with
```
export ELECTRON_MIRROR="https://mirrors.huaweicloud.com/electron"
export NODE_MIRROR="https://nodejs.org/dist"
```
and it says
```
info DIST Downloading distribution files.
http DIST       - https://mirrors.huaweicloud.com/electron/v9.2.1/node-v9.2.1.tar.gz
ERR! OMG incorrect header check
ERR! OMG incorrect header check
```

**Comment by Julusian** [comment]
The electron url was updated to `https://artifacts.electronjs.org/headers/dist` in v6.3.2 of cmake-js.

Updating to that version or the later 7.x releases will resolve this

**Comment by kistlin** [comment]
Yeah true, I was in an old backup folder. But the version used now is `cmake-js@7.1.1`.
So there must be something else going on.
